plugin/storage/integration/cassandra_test.go

```diff
@@ -67,27 +65,14 @@ func (s *CassandraStorageIntegration) initializeCassandra(t *testing.T) {
 		"--cassandra.username=" + username,
 		"--cassandra.keyspace=jaeger_v1_dc1",
 	}, cassandra.NewFactory)
-	af := s.initializeCassandraFactory(t, []string{
-		"--cassandra-archive.keyspace=jaeger_v1_dc1_archive",
-		"--cassandra-archive.enabled=true",
-		"--cassandra-archive.servers=127.0.0.1",
-		"--cassandra-archive.basic.allowed-authenticators=org.apache.cassandra.auth.PasswordAuthenticator",
-		"--cassandra-archive.password=" + password,
-		"--cassandra-archive.username=" + username,
-	}, cassandra.NewArchiveFactory)
```

Comment:
@yurishkuro thoughts on keeping vs removing this? we were initializing the archive span reader / writer but never actual ran the archive test

Reply:
ok